libraries/lib-datahandler/src/main/thrift/projects.thrift

```diff
@@ -85,6 +85,7 @@ struct Project {
 //    26: optional set<string> comoderators, //deleted
     27: optional set<string> contributors = [],
     28: optional Visibility visbility = sw360.Visibility.BUISNESSUNIT_AND_MODERATORS,
+    129: optional set<string> securityResponsibles = [],
```

Comment:
I am not sure whether the default value for this optional field is necessary / positive since one can not rely on it.